server/index.js

```diff
@@ -7,8 +7,18 @@ const passport = require('passport');
 const cookieParser = require('cookie-parser');
 const session = require('express-session');
 const path = require('path');
+/* eslint-disable import/order */
```

Comment:
Why this rule is disabled? Wouldn't it be better to just sort the imports?

Reply:
Added as EOC-433 task in JIRA